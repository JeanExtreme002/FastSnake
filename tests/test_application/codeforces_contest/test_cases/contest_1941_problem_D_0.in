5
6 3 2
2 ?
2 ?
2 ?
12 1 2
3 1
10 7 4
2 ?
9 1
4 ?
7 0
2 0
8 1
5 ?
5 3 1
4 0
4 ?
1 ?
4 1 1
2 ?