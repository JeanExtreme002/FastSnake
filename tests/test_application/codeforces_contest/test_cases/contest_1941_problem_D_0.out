3
2 4 6 
1
11 
4
3 5 7 9 
3
2 3 5 
1
3